README.md

```diff
@@ -12,13 +12,9 @@ PermCheck aims to prevent inconsistent executable bits and thus random executabl
 
 ## Installation
 
-PermCheck can be added as a composer package by adding it to your composer.json:
+PermCheck can be added as a composer package by running:
 
-    {
-        "require-dev": {
-            "existenznl/permcheck": "^1.0"
-        }
-    }
+    composer require --dev existenznl/permcheck
```

Comment:
Great! This was something I had on my list for a long time but again and again forgot to fix. Thx for the catch!